### PR TITLE
 Insertion Sort - by LearnerBoy

### LINKED ISSUE #1: Missing Insertion Sort code
Make a PR with C++ code for Insertion Sort.

Try to minimize the time complexity as much as possible.

### PR DESCRIPTION
**Pull request by ayanbiswas1101**
Fixes #1 
Insertion sort code. @jddeep Please review.

### CODE REVIEW

**Comment by ayanbiswas1101** [comment]
I have added the Insertion Sort algorithm using C++. The given code has successfully run in Dev C++ compiler. The work has done #1 issue

File: Lboy [hunk 1]
```diff
@@ -0,0 +1 @@
+snoop dogg di
```
Comment (jddeep) [inline Lboy]: @ayanbiswas1101 Please remove this file from the PR. Thanks

**Review by jddeep — CHANGES_REQUESTED** [review]

**Comment by jddeep** [comment]
> I have added the Insertion Sort algorithm using C++. The given code has successfully run in Dev C++ compiler. The work has done #1 issue

@ayanbiswas1101 Always direct issue fixes in the PR body description. I have done it this first time. Check the PR description part. Follow that.

**Review by jddeep — APPROVED** [review]
Good job :+1:

**Pull request merged**